pkg/update/update.go

```diff
@@ -30,6 +30,15 @@ func DisplayAndCheckForRelease() error {
 	if err != nil {
 		return err
 	}
+
+	if state.Version != global.Version {
+		console.Debugf("Resetting update message because Cog has been upgraded")
+		state.Message = ""
```

Comment:
Should we not also set `state.Version = global.Version` here?

Reply:
It's a no-op because no message means it doesn't do anything but I suppose we may as well.

Reply:
Ah, yes we should otherwise it'll rewrite the config on each command until the next version check.